### ISSUE TITLE
Type not interpret when using `include` at top level

### ISSUE THREAD

**Issue by hima2nd** [castwide/solargraph#170]
VSCode : Ruby Solargraph Version 0.19.3
Solargraph Version 0.31.3

Here's example code:

```
module A
  module B
    module C
      def self.hoge; end
    end
  end
end

include A
B::C. # <- not interpret type A::B::C

```

**Comment by castwide** [comment]
Confirmed. I'll work on a fix.

**Comment by castwide** [comment]
The fix is in the master branch. It'll be released in version 0.32.0.

**Comment by castwide** [comment]
Released in v0.32.0.
